src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,49 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+<<<<<<< HEAD
```

Comment:
This is left from git conflict resolution, please remove.

Reply:
done

Reply:
done